eng/common/pipelines/templates/steps/verify-links.yml

```diff
@@ -14,4 +14,5 @@ steps:
         -urls $(dir -r -i *.md) 
```

Comment:
Can you also add parameters for the workingDirectory and scriptDirectory similar to what we do in create-pull-request.yml file at https://github.com/Azure/azure-sdk-tools/blob/master/eng/common/pipelines/templates/steps/create-pull-request.yml#L59.